devices.ttl

```diff
@@ -683,11 +683,20 @@ _:boss-rc-10r device:typeA true .
 _:boss-rc-10r device:uri "https://www.boss.info/global/products/rc-10r" .
 
 _:retrokits-RK-006 device:make "Retrokits" .
-_:retrokits-RK-006 device:model "RK-006" .
+_:retrokits-RK-006 device:model "RK-006 Hub" .
```

Comment:
I prefer to have the `model` field be strictly the model number or identifier, and describe that model in the `notes` field. For example, a `notes` field like "Portable USB MIDI/Gate Hub" would be appropriate.